README

```diff
@@ -765,6 +765,22 @@ Open MPI is unable to find relevant support for <foo>, configure will
 assume that it was unable to provide a feature that was specifically
 requested and will abort so that a human can resolve out the issue.
 
+Additionally, if a search directory is specified in the form
+--with-<foo>=<dir>, Open MPI will:
```

Comment:
Given the user that reported this was using ```--with-foo-libdir```, you probably should include that here as well

Reply:
Will add #5773 when CI completes.